### DISCUSSION TITLE
Add European Qualifications Framework (EQF) property

### DISCUSSION THREAD

**Discussion by KenthJohan** [schemaorg/schemaorg#3044] [Ideas]
Can we have EQF property?
https://en.wikipedia.org/wiki/European_Qualifications_Framework

Can EQF be added for https://schema.org/Course?

**Comment by thadguidry** [comment]
@philbarker don't you know a few folks in the commission on EQF?  Where was the alignment decided for? Course?

**Comment by philbarker** [comment]
EQF Levels can be used with the `educationalLevel` property as `DefinedTerm`s in the same way that SQF level is shown in [the example](https://schema.org/educationalLevel#eg-0259).
